vpr/src/place/place.cpp

```diff
@@ -1840,31 +2163,21 @@ static void revert_td_cost(const t_pl_blocks_to_be_moved& blocks_affected) {
 //
 //Relies on proposed_connection_delay and connection_delay to detect
 //which connections have actually had their delay changed.
```

Comment:
Explain purpose of the routine (to figure out which connections have had their delays changed so we can incrementally timing analyze).  Do we need to call this after every committed move or every proposed move?  Explain a bit more about its use here.